docs/monitors/_monitor-config.md

```diff
@@ -0,0 +1,174 @@
+
+<!--- Generated by to-integrations-repo script in Smart Agent repo, DO NOT MODIFY HERE --->
+<!--- GENERATED BY gomplate from scripts/docs/templates/monitor-main.md.tmpl --->
+
+# Monitor Configuration
+
+Monitors gather metrics from the host and from running applications.  They are
```

Comment:
There appears to be an extra space before the second sentence.

Reply:
I didn't change this file. I'm not going to touch it.